money-api/src/test/scala/com/comcast/money/api/DefaultSpan.scala

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2012-2015 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.comcast.money.api
+
+import java.lang
+import java.lang.{ Double, Long }
+
+import scala.collection._
+import scala.collection.JavaConversions._
+
+// obviously not threadsafe and very mutable, basing this kinda on what I did for the java-only core
+class DefaultSpan(
+    val id: SpanId,
+    val spanName: String,
+    spanMonitor: SpanMonitor,
+    notes: mutable.Map[String, Note[_]] = new mutable.HashMap[String, Note[_]],
+    propagate: Boolean = false
+) extends Span {
+
+  // TODO: yea, hacky, just prototyping though
+  private var stopTime: Long = _
+  private var stopInstant: Long = _
+  private var startedTime: Long = _
+  private var startInstant: Long = _
+  private var success: Boolean = true
+
+  override def startTime(): Long = startedTime
+
+  override def start(): Unit = {
+    startedTime = System.currentTimeMillis
+    startInstant = System.nanoTime
+
+    // Start watching me
+    spanMonitor.watch(this)
```

Comment:
Let's discuss more what the default implementation for Span management should be.  Ideally most users should not be burdened with having to weigh the merits of `money-akka` vs. `money-scala` vs. `money-scala` vs `money-java`  

I propose we end up with a single dependency that include the Java API as you defined, A scala implementation of Span and Span Handler, and a scala implementation of Span management that may or not be Akka based depending on whether we think the benefits we get from Akka are worth the potential issues we may run into upgrading Akka versions and/or dealing projects that already include Akka.  That is a discussion we can delay for now as I don't think its a blocker for ironing out the API.  Although it probably makes sense to stick w/ the status quo for now.

There will be another library for those who want to manage Spans on their own.  This would include the java-api as well as the scala implementation of Span and maybe some SpanHandlers.  Or maybe its just the java-api.

And finally there will be a really slim library money-micro (or whatever) that includes the java-api a java-implementation of Span and SpanHandler and a java implementation of SpanManagement without Akka.  That could totally end up being pretty much what you have here in this patch set.